include/options/options.php

```diff
@@ -116,13 +108,7 @@ public function register( string $class_name, string $key, $default, string $des
 			}
 
 			// Option raw value exists in database, use it.
-			$options[ $key ] = new $class_name(
-				$key,
-				$options[ $key ],
-				$default,
-				$description,
-				...$args
-			);
+			$options[ $key ] = new $class_name( $options[ $key ] );
```

Comment:
I think it could be written only once with an improved condition around line 98, what do you think?

Reply:
Like this?

```php
if ( ! array_key_exists( $key, $options ) || ! $options[ $key ] instanceof Abstract_Option ) {
	$options[ $key ] = new $class_name( $options[ $key ] ?? null );
}
```

I'm not sure to like it.

Reply:
Yup! But it's a matter of taste, you get to choose 😉